pages/design/rule-design.md

```diff
@@ -63,6 +63,21 @@ The description is a short synopsis of what the rule does. This should be declar
 
 List all accessibility requirements that are not satisfied if one or more outcomes of a rule fails. For example, this could be one or more WCAG success criteria, or a WAI-ARIA rule. Each requirement should indicate the associated conformance level, and map the implications of each outcome, such as "satisfied", "not satisfied", or "further testing needed".
 
+Accessibility requirements should be listed as secondary if the requirement is related to the rule. There are three scenarios:
+
+1. The requirement is less strict than the rule. If this occurs, and some but not all failed examples of the rule pass the accessibility requirement, the following test should be used:
+
+   > This success criterion is **less strict** than this rule. This is because {reason}. Some of the failed examples satisfy this success criterion.
+
+1. The requirement is more strict than the rule. If this occurs, and some but not all passed examples of the rule fail the accessibility requirement, the following test should be used:
+
+   > secondary: This success criterion is **more strict** than this rule. This is because {reason}. Some of the passed examples do not satisfy this success criterion.
+
+1. The requirement is related to, but different from the rule. This often happens when there is overlap between two accessibility requirements. If this occurs, and some but not all the failed examples of the rule fail the accessibility requirement, the following text should be used:
+   > This success criterion is **related** to this rule. This {is because / happens for} {reason}. Most failed examples satisfy this success criterion.
+
+Secondary requirements are optional, and are only necessary when fails some of the passed examples, or passes some of the failed examples. If there are no such examples, it is generally not necessary to include secondary requirements. Atomic rules used in composite rules should list the requirements from the composite rule as "more strict" secondary requirements.
```

Comment:
```suggestion

1. Atomic rules used in composite rules should list the requirements from the composite rule as "more strict" secondary requirements.

Secondary requirements are optional and are only necessary when a requirement fails some of the passed examples or passes some of the failed examples. If there are no such examples, it is generally not necessary to include secondary requirements. 
```

I like the last sentence and think it needs more attention, so suggest making it one of the list items.